api/base_controller.go

```diff
@@ -195,6 +203,30 @@ func (c *BaseController) GetSingleObject(r *web.Request) (*web.Response, error)
 	return util.NewJSONResponse(http.StatusOK, object)
 }
 
+// GetOperation handles the fetching of a single operation with the id specified for the specified resource
+func (c *BaseController) GetOperation(r *web.Request) (*web.Response, error) {
```

Comment:
Should we implement ListOperation for given objectID? For example: operations for all brokers?

Reply:
Not needed for the async engine part (it will be introduced for admin purposes eventually when we get there i guess ?). And for admin purposes i see it as a too level list operations api secured with its own operations scopes as we dont wqnt to hand it out in the location header